index.js

```diff
@@ -1,35 +1,64 @@
-var express = require("express");
+var express = require('express');
 var app = express();
-var bodyParser = require("body-parser");
+var bodyParser = require('body-parser');
 var jsonParser = bodyParser.json({ extended: false });
-var mongoose = require("mongoose");
-var uristring = "mongodb://localhost:27017/appointus";
-var Client = require("./models/clients");
+var mongoose = require('mongoose');
+mongoose.set('useFindAndModify', false);
+mongoose.set('useCreateIndex', true);
+var uristring = 'mongodb://localhost:27017/appointus';
+var Client = require('./models/clients');
 var Appointment = require("./models/appointments");
 
 mongoose.connect(uristring, { useNewUrlParser: true }, function(err) {
-  if (err) {
-    console.log(err);
-  }
+	if (err) {
```

Comment:
Different space size. It is needed to sync it in team.

Reply:
done